src/@components/MainPage/Banner/style.ts

```diff
@@ -113,7 +113,7 @@ export const CurrentPage = styled.span`
   color: ${({ theme }) => theme.newColors.white};
 `;
 
-export const Gradient = styled.div`
+export const Gradient = styled.div<{ isLightMode: boolean }>`
```

Comment:
스타일드 컴포넌트 prop명은 전체 소문자로 해야 어떤 워닝이 피해지던 경험이 있더라구요.?
```suggestion
export const Gradient = styled.div<{ islightmode: boolean }>`
```

Reply:
헐 몰랐네요 좋은 정보 감사합니다!